docs/UI/ScrollableMessage/index.md

```diff
@@ -19,6 +19,7 @@ Purpose
 |timeout|Integer|true|minvalue: 0<br>maxvalue: 65535|
 |softButtons|[Common.SoftButton](../../common/structs/#softbutton)|false|array: true<br>minsize: 0<br>maxsize: 8|
 |appID|Integer|true||
+|cancelID|Integer|false|can be used later with CancelInteraction RPC to specifically close this scrollable message|
```

Comment:
This table does not have a description section

Reply:
fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/1f93da0ff41d60f00c4d1bb3cc93ffeb5bd034e0